### PR TITLE
update composer.json to ~2.4

### PR DESCRIPTION
**Pull request by samsonasik**
based on http://framework.zend.com/long-term-support

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
     "homepage": "http://framework.zend.com/",
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zendframework": "~2.3"
+        "zendframework/zendframework": "~2.4.0"
```
Comment (Ocramius) [inline composer.json]: `~2.4`

**Comment by samsonasik** [comment]
@Ocramius done ;)

**Comment by samsonasik** [comment]
@Ocramius  @weierophinney  merge-able ?

**Comment by Ocramius** [comment]
:+1:

**Comment by samsonasik** [comment]
@Ocramius when using ~2.4, I got 2.5.1, it is that purpose ? or we should use ~2.4.0 instead ?

**Comment by Ocramius** [comment]
@samsonasik `~2.4` means `>=2.4,<3.0`

**Comment by samsonasik** [comment]
ok

**Comment by DonCarlosGraz** [comment]
hi! 

I noticed that there is no 2.4.x package on 
https://packages.zendframework.com/

Is this planned in the near future?

Thanks in advance

**Comment by Maks3w** [comment]
@DonCarlosGraz Please open your issue in https://github.com/zendframework/ZF2Package

**Pull request merged**